src/main/kotlin/no/nav/syfo/apprec/model/ApprecStatus.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.syfo.apprec.model
+
+enum class ApprecStatus(val v: String, val dn: String) {
+    AVVIST("2", "Avvist"),
+    OK("1", "OK"),
+}
```

Comment:
De ulike typene er vel BESTILT, SENDT, OK, AVVIST, men disse to holder forsåvidt for vår testcase.

Reply:
I denne sammenhengen så gir ikke BESTILT og SENDT noen mening tror jeg. Det blir en forskjell på status på dialogmeldingbestillingen og status på apprec'en som (kanskje) mottas etter at bestillingen er sendt.

Reply:
For Apprecs er det streng tatt bare OK eller AVVIST. Så har vi utvidet med BESTILT og SENDT som statuser i `isdialogmelding`, men disse trenger man ikke gå via isyfomock for å få testet.